data_subscriptions/notifications/email_template.py

```diff
@@ -49,7 +49,11 @@ class DatasetActivity:
     def __init__(self, dataset, activities):
         self.dataset = dataset
         self.activities = activities
-        self.ckan_api = RemoteCKAN(CKAN_URL, apikey=CKAN_API_KEY)
+        self.ckan_api = RemoteCKAN(
+            CKAN_URL,
+            apikey=CKAN_API_KEY,
+            user_agent="data-subscription/latest (API call for CKAN activity details)",
```

Comment:
@sagargg I think my suggestion was "API call for CKAN package activity details"

Reply:
@anuveyatsu  This specific API returns activity details for an activity Id.  I'm adding a relevant comment with its action name `acitivity_detail`.  **Note:** it will not a resource/package level action. It returns other activity as well.

Reply:
@sagargg got it, thanks for clarification 👍